### PR TITLE
howdoi 1.1.12 (new formula)

### PR DESCRIPTION
**Pull request by emanresusername**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
could only test on linuxbrew fork, but [sounds like it worked on mac](https://github.com/gleitz/howdoi/pull/174#issuecomment-347066482) (although this was modified according to the audit, so not exactly the same)

### CODE REVIEW

**Comment by emanresusername** [comment]
not sure what's up with the sierra build? :confused:

**Comment by ilovezfs** [comment]
It's an lxml bug. Set SDKROOT and you should be OK. See

e15fcfc201d7be432073a04daaf2d515c13c214c
7794a60224ba202a91b025fe3ab6666ada2461c3

**Comment by emanresusername** [comment]
Un-stale-ify

**Comment by vinyldarkscratch** [comment]
@BrewTestBot test this please

**Comment by vinyldarkscratch** [comment]
Seems that `howdoi` has its own tap and formula, with lots of updates (version number and URL).  Test bot isn't liking the current URL in this PR anymore.  Can you grab the formula from https://raw.githubusercontent.com/gleitz/howdoi/master/howdoi.rb real quick?

**Comment by vinyldarkscratch** [comment]
Thank you!

File: Formula/howdoi.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
+class Howdoi < Formula
+  include Language::Python::Virtualenv
+
+  desc "Instant coding answers via the command-line"
+  homepage "https://github.com/gleitz/howdoi/"
+  url 'https://pypi.python.org/packages/67/e0/18ca30a65aeed25c6d6fe7171d71601b3a772bf0b7b8235b9e98d3cb889e/howdoi-1.1.12.tar.gz'
```
Comment (vinyldarkscratch) [inline Formula/howdoi.rb]: The audit is saying that this line should read exactly like:

`url "https://files.pythonhosted.org/packages/67/e0/18ca30a65aeed25c6d6fe7171d71601b3a772bf0b7b8235b9e98d3cb889e/howdoi-1.1.12.tar.gz"`

File: Formula/howdoi.rb [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  sha256 'c8676cea5155a8bb3d996f3555942e2726029338c606e36ed32775717a5e7bac'
+
+  depends_on :python if MacOS.version <= :snow_leopard
```
Comment (vinyldarkscratch) [inline Formula/howdoi.rb]: Can you replace `:python` with `python`?

**Review by queengooborg — CHANGES_REQUESTED** [review]

File: Formula/howdoi.rb [hunk 3]
```diff
@@ -0,0 +1,57 @@
+class Howdoi < Formula
+  include Language::Python::Virtualenv
+
+  desc "Instant coding answers via the command-line"
+  homepage "https://github.com/gleitz/howdoi/"
+  url 'https://pypi.python.org/packages/67/e0/18ca30a65aeed25c6d6fe7171d71601b3a772bf0b7b8235b9e98d3cb889e/howdoi-1.1.12.tar.gz'
+  sha256 'c8676cea5155a8bb3d996f3555942e2726029338c606e36ed32775717a5e7bac'
```
Comment (vinyldarkscratch) [inline Formula/howdoi.rb]: Oops, missed one: can you replace the single quotes with double quotes?

**Review by queengooborg — CHANGES_REQUESTED** [review]

**Comment by vinyldarkscratch** [comment]
@emanresusername The URL that you’ve currently got redirects to `https://files.pythonhosted.org/packages/67/e0/18ca30a65aeed25c6d6fe7171d71601b3a772bf0b7b8235b9e98d3cb889e/howdoi-1.1.12.tar.gz` — can you set the URL to this, please?

**Review by queengooborg — APPROVED** [review]
Looks all good now -- thank you, @emanresusername!

**Comment by vinyldarkscratch** [comment]
@ilovezfs Ready for merge?

**Review by ilovezfs — APPROVED** [review]

**Comment by ilovezfs** [comment]
Thanks for the :new: formula @emanresusername!

**Pull request closed without merging**